packages/client/tests/functional/logging-mongo/tests.ts

```diff
@@ -0,0 +1,73 @@
+import testMatrix from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+// @ts-ignore this is just for type checks
+type PrismaClient = import('@prisma/client').PrismaClient
+// @ts-ignore this is just for type checks
+declare let newPrismaClient: NewPrismaClient<PrismaClient>
+
+testMatrix.setupTestSuite(
+  () => {
+    test('basic event logging', async () => {
+      const prisma = newPrismaClient({
+        log: [
+          {
+            emit: 'event',
+            level: 'query',
+          },
+        ],
+      })
+
+      const onQuery = jest.fn()
+
+      prisma.$on('query', onQuery)
+
+      await prisma.user.findMany()
+
+      expect(onQuery).toHaveBeenCalledWith({
+        params: expect.any(String),
+        query: expect.any(String),
+        target: 'mongodb_query_connector::query',
```

Comment:
```suggestion
        target: expect.any(String),
```
Is it worth doing any for this? They could change it engine side and it will force us to update tests.